src/main/resources/view/PersonListCard.fxml

```diff
@@ -9,32 +9,66 @@
 <?import javafx.scene.layout.Region?>
 <?import javafx.scene.layout.RowConstraints?>
 <?import javafx.scene.layout.VBox?>
+<?import javafx.scene.control.CheckBox?>
+
+
+<HBox fx:id="cardPane" 
+    xmlns="http://javafx.com/javafx/8" 
+    xmlns:fx="http://javafx.com/fxml/1">
+    <GridPane HBox.hgrow="ALWAYS">
```

Comment:
What is the gridpane for exactly here? Is it just to add the extra margin/padding?

Reply:
Yes for the margin